src/components/dataset/summary/row.vue

```diff
@@ -149,9 +149,9 @@ export default {
 <i18n lang="json5">
 {
   "en": {
-    // This is shown when a dataset is new
+    // This is shown when an Entity List is new
     "new": "new!",
```

Comment:
If the noun has changed, does "new!" need to be retranslated? You could rename the key (e.g., to `new2` or `newEntityList`) to force retranslation.

Reply:
I think we should take our chances. None of the languages I know would require a change. Maybe German? I'm going to send out a message to translators anyway so I can highlight that.